python/tests/unit/contents/test_chat_history.py

```diff
@@ -315,6 +317,30 @@ async def test_template(chat_history: ChatHistory):
     assert chat_history_2.messages[2].role == AuthorRole.USER
 
 
+@pytest.mark.asyncio
+async def test_template_safe(chat_history: ChatHistory):
+    chat_history.add_assistant_message("I am an AI assistant")
+
+    template = "system stuff{{$chat_history}}{{$input}}"
```

Comment:
Does python have special handling for `ChatHistory` instances?

Reply:
yes, we added a to_string method to chat_history in order to parse it into a XML string, and back, this unifies the ability to have a chat_history param in all three template languages, @matthewbolanos did mention that this should be backported to dotnet at some point